scripts/src/internal/jwt.ts

```diff
@@ -0,0 +1,47 @@
+import * as moment from "moment";
+import { readFileSync } from "fs";
+import * as jsonwebtoken from "jsonwebtoken";
+
+import { getConfig } from "./config";
+
+const CONFIG = getConfig();
+
+type KeyMap = Map<string, { algorithm: string; key: Buffer; }> & {
+	random(): { id: string; key: Buffer; algorithm: string; };
+};
+const KEYS = (() => {
+	const map = new Map<string, { algorithm: string; key: Buffer; }>() as KeyMap;
```

Comment:
`new KeyMap` doesn't work?

Reply:
`KeyMap` is a type, it only exists in the compilation time, you cannot instantiate it at runtime